sphinx_immaterial/apidoc/apidoc_formatting.py

```diff
@@ -96,7 +96,8 @@ def depart_caption(
         "literal_block"
     ):
         self.body.append("</div>\n")
-    else:
+    elif not isinstance(node.parent, docutils.nodes.figure):
+        # calling super_func() on a figure results in 2 permalinks in caption
```

Comment:
I assume this is an independent fix?

Reply:
Yeah, I noticed this bug when using a figure to caption an image in the new doc. I can submit this an a separate PR though, if desired.

Reply:
It is okay to combine it, just wanted to confirm.